### PR TITLE
Form Field being disabled issue

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
# Issue Tracking

JIRA: FWF-874

# Changes

- Created a separate component to call formio for edit/View mode
- On task claiming the `FormEdit` component is activated and on readonly`FormView`  component is called.

# How has the change been tested?
- On testing in app2vue, the issue was not reported again

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: camunda-formio-tasklist-vue/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "camunda-formio-tasklist-vue",
-  "version": "1.0.11",
+  "version": "1.0.13",
```
Comment (ann-aot) [inline camunda-formio-tasklist-vue/package.json:3]: 👍

File: camunda-formio-tasklist-vue/src/components/TaskList.vue [hunk 2]
```diff
@@ -371,20 +368,21 @@ const StoreServiceFlowModule = namespace("serviceFlowModule");
...
     BpmnViewer,
+    FormEdit,
+    FormView
   },
 })
 export default class Tasklist extends Mixins(TaskListMixin) {
   @Prop() private getTaskId!: string;
   @Prop() private containerHeight!: string;
-  @Prop({ default: "lastName" }) userListType!: string;
+  //@Prop({ default: "lastName" }) userListType!: string;
```
Comment (ann-aot) [inline camunda-formio-tasklist-vue/src/components/TaskList.vue:385]: we can delete the comment

**Review by ann-aot — APPROVED** [review]

File: camunda-formio-tasklist-vue/src/components/TaskList.vue (hunk 2 above)
Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/TaskList.vue:385]: Sure 👍

**Comment by kurianbenoy-aot** [comment]
Merging #87

**Pull request merged**